jinahub/indexers/cache/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM jinaai/jina:1.0.2
+
+# install the third-party requirements
+RUN apt-get update && apt-get install --no-install-recommends -y gcc build-essential git
+
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
+
+RUN pip install -r requirements.txt
+
+# for testing the image
+RUN pip install pytest && pytest -s -v tests
+
+ENTRYPOINT ["jina", "pod", "--uses", "config.yml"]
```

Comment:
I believe this should become `..."jina", "pea", ...`, but it might anyhow change completely when JinaD 2.0 is finished. Opinions @deepankarm ?

Reply:
it should be jina pea I agree, but the sugar of using executor would make sense

Reply:
This needs to be decided in core & JinaD would adapt to it. With JinaD 2.0, we still support containerized pods & handle them by just modifying host and mapping ports wherever required.